.github/workflows/unit_tests.yml

```diff
@@ -175,7 +175,7 @@ jobs:
         pushd ..
         # Get npm 7+
         npm install -g npm
-        git clone --depth 1 https://github.com/bids-standard/bids-validator
+        git clone --depth 1 --branch nirs https://github.com/rob-luke/bids-validator.git
```

Comment:
Revert this before merge!

Reply:
see: https://github.com/mne-tools/mne-bids/pull/406#issuecomment-1057762831